extensions/virtuallistview/src/VirtualListView.tsx

```diff
@@ -1010,8 +1010,11 @@ export class VirtualListView<ItemInfo extends VirtualListViewItemInfo>
         // Set the "live" values as well.
         let cell = this.refs[cellInfo.virtualKey] as VirtualListCell;
         if (cell) {
+            // on screen when full message is within layoutHeight area.
+            let isOnScreen = top > this._lastScrollTop && (top + cellInfo.height) < this._lastScrollTop + this._layoutHeight;
```

Comment:
This check requires that the item be on screen in its entirety, as opposed to partly on screen. It's not clear to me that's what all callers will want. It depends on the use case.